egs/multi_en/s5/local/make_partitions.sh

```diff
@@ -0,0 +1,68 @@
+#!/bin/bash
+
+# Copyright 2016  Allen Guo
+# Apache License 2.0
+
+multi=multi_a
```

Comment:
Please describe what the parameter is, what the accepted values are and what they mean.